### ISSUE TITLE
Avian 4.1 release plan

### ISSUE THREAD

**Issue by alamshafil** [AvianNetwork/Avian#169]
This issue tracks the work that needs to be done to release Avian 4.1. 
Nothing here is final and is subject to **CHANGE**!

**Tentative deadlines**

- [x] Deadline for pull requests: 2022-09-23

**MUST HAVE (delay without these having been merged)**

- [x] #87 
  - [x] Set main-net address (multi-sig)
  - [x] Clean up debug log `FounderPayment::FillFounderPayment -- Founder payment 0`
  - [x] Fix testnet
- [x] #162
- [x] #164
- [x] #168
- [x] #170 
- [x] #172 

**COULD HAVE (non-critical for release)**

- [ ] #72 
- [x] #157 
- [x] #159 
- [x] #171 

**Maintainer tasks for release**

- [x] Update fixed seeds #175 
- [x] Update checkpoints #178 
- [x] Update copyrights
- [x] Write release notes #176 
- [x] Mark v4.1 as release in source #179
- [x] Tag v4.1
- [x] Builds for v4.1
- [x] Update master
- [x] Upload binaries and sha256sum files

**Comment by alamshafil** [comment]
4.1 is ready for release, building binaries now.

**Comment by alamshafil** [comment]
4.1 has been released!

**Comment by alamshafil** [comment]
To properly assist you, make sure you running Avian 4.1 on commit https://github.com/AvianNetwork/Avian/commit/53947c4d7aeeb4bbd0fd421517f15dd0827e3979.

On my local node I was able to run the below without any issues:

- `getblocktemplate`
- `getblocktemplate "{\"powalgo\": \"minotaurx\"}"`
- `getblocktemplate "{\"powalgo\": \"x16rt\"}"`

Keep in mind that dev fee is already subtracted from `coinbasevalue`, ex: `237500000000`.

From the value `262500000000`, it looks like 5% of 2500 (125) was added to coinbase subsidy of 2500 (2625).

It is odd that your gbt is adding to coinbase. This is the code that checks coinbase: *(`AreEnforcedValuesDeployed()` is always true)*
https://github.com/AvianNetwork/Avian/blob/53947c4d7aeeb4bbd0fd421517f15dd0827e3979/src/validation.cpp#L2805-L2811

I know this is not related to getting a block template but make sure your pool is properly configured to support the dev fee. (Keep in mind that the founder address is a P2SH multi-sig addr and do not subtract from coinbase.)
Yiimp example: https://github.com/mivanoski/yiimp/commit/9cff8299921611c83f7787632e7aa76519b97077 

If you are still experiencing this problem, make a new issue with additional information (if possible). I apologize for the inconvenience.
